src/status/components/StatusIndicator.tsx

```diff
@@ -4,5 +4,16 @@ import { State } from '../ServerStatus';
 export type Props = ThemedStyledProps<State, DefaultTheme>;
 
 export const StatusIndicator = styled.p<Props>`
-  color: ${(props: Props) => props.status === 'alive' ? props.theme.colors.green : props.theme.colors.red}
+  align-self: center;
+  height: 25px;
+  width: 25px;
+  border-radius: 50%;
+  background-color: ${(props: Props) => props.status === 'alive' ? props.theme.colors.green : props.theme.colors.red};
+  animation: fadein 2s infinite;
+
+  @keyframes fadein {
```

Comment:
`fadeIn` and `fadeOut` can be used as mixin

Reply:
done